DigitalLearningSolutions.Web/babel.config.js

```diff
@@ -7,7 +7,7 @@
             targets: {
               ie: '11',
             },
-            corejs: '3.6',
+            corejs: '3',

```

Comment:
Because we were already on 3.8.1, so 3.6 was the wrong version, and I updated to 3.22.5. However, I was misremembering the documentation, which states that you _should_ give a minor version.

I will fix this so that it reads the version from package.json and uses that.